newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -794,6 +825,152 @@ export const LeaderboardAdmin = ({
         </IconButton>
       ),
     },
+    {
+      key: 'ignoreCustomPlayerNames',
+      avatar: currentLeaderboard.ignoreCustomPlayerNames ? (
+        <Lock />
+      ) : (
+        <LockOpen />
+      ),
+      text: (
+        <Tooltip
+          title={i18n._(
+            currentLeaderboard.ignoreCustomPlayerNames
+              ? t`Even if the action is used to send a score with a custom player username, this name will be ignored by the leaderboard.`
+              : t`The player name sent in the action to send a score will be used.`
+          )}
+        >
+          <Text size="body2">
+            {currentLeaderboard.ignoreCustomPlayerNames ? (
+              <Trans>Ignore unauthenticated player usernames</Trans>
+            ) : (
+              <Trans>Allow unauthenticated player usernames</Trans>
+            )}
+          </Text>
+        </Tooltip>
+      ),
+      secondaryText:
+        apiError &&
+        apiError.action === 'leaderboardIgnoreCustomPlayerNamesUpdate' ? (
+          <Text color="error" size="body2">
+            {apiError.message}
+          </Text>
+        ) : null,
+      secondaryAction: (
+        <IconButton
+          onClick={async () => {
+            await onUpdateLeaderboard(i18n, {
+              ignoreCustomPlayerNames: !currentLeaderboard.ignoreCustomPlayerNames,
+            });
+          }}
+          tooltip={
+            currentLeaderboard.ignoreCustomPlayerNames
+              ? t`Change to ignore custom player usernames`
+              : t`Change to allow custom player usernames`
+          }
+          edge="end"
+          disabled={isRequestPending || isEditingName}
+        >
+          <SwitchHorizontal />
+        </IconButton>
+      ),
+    },
+    {
+      key: 'autoPlayerNamePrefix',
+      avatar: <Tag />,
+      text: isEditingAutoPlayerNamePrefix ? (
+        <Line alignItems="center" expand noMargin>
+          <TextField
+            id="edit-autoPlayerNamePrefix-field"
+            ref={newAutoPlayerNamePrefixTextFieldRef}
+            margin="none"
+            style={styles.leaderboardNameTextField}
+            maxLength={50}
```

Comment:
To match the backend limit
```suggestion
            maxLength={40}
```